authorizer.py

```diff
@@ -142,10 +147,8 @@ def Redirect(self):
     logging.info("Redirecting %s to /welcome", self.email)
     self.handler.redirect("/welcome")
 
-  # TODO get rid of the unnecessary handler parameter
```

Comment:
There is no harm as far as I know. This is just trivia. There is no reason
to change it now.

FYI: you could write

auth = authorizer.Authorizer()

...because the WebApp2 framework provides a way to get the current request.
But again, this is not important.
On Jan 28, 2016 6:46 PM, "s-moffatt" <notifications@github.com> wrote:

> In authorizer.py
> <https://github.com/arthurw9/selectives-web/pull/21#discussion_r51220282>:
>
> > @@ -142,10 +147,8 @@ def Redirect(self):
> >      logging.info("Redirecting %s to /welcome", self.email)
> >      self.handler.redirect("/welcome")
> >
> > -  # TODO get rid of the unnecessary handler parameter
>
> Not just for GetLogoutUrl, but the entire authorizer?
> Do you mean initialize authorizer on every page
> this way
> auth = authorizer.Authorizer(self.request)
> instead of
> auth = authorizer.Authorizer(self)
>
> What is the harm in passing the handler? Is it simply not necessary or is
> there real harm (security risk or something)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arthurw9/selectives-web/pull/21/files#r51220282>.
>